### PR TITLE
Move `e2e-aws-csi` to multi-stage

### PR DESCRIPTION
**Pull request by bbguimaraes**

### CODE REVIEW

**Comment by bbguimaraes** [comment]
/hold

Will require changes to `ci-operator`.

File: ci-operator/step-registry/openshift/e2e/aws/csi/OWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+approvers:
+- smarterclayton
+- wking
+- stevekuznetsov
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/openshift/e2e/aws/csi/OWNERS:4]: Can we use a smaller list here?

Comment (bbguimaraes) [reply ci-operator/step-registry/openshift/e2e/aws/csi/OWNERS:4]: Hmm, this was supposed to be a symlink to `../OWNERS` as in the other PRs, not sure what happened (the contents are the same).

The original tests don't have any ownership information, so I don't know what that list should be.

**Comment by bbguimaraes** [comment]
Successful execution: https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/9922/rehearse-9922-pull-ci-openshift-origin-master-e2e-aws-csi/1280557757052751872.

File: ci-operator/step-registry/openshift/e2e/aws/csi/OWNERS (hunk 1 above)
Comment (bbguimaraes) [reply ci-operator/step-registry/openshift/e2e/aws/csi/OWNERS]: Replaced it with a symlink.

**Comment by bbguimaraes** [comment]
I'll monitor it.

/hold cancel

**Pull request merged**